Docs 3398: Readme formatting (#2269)
MultiSafepay/docs: content/integrations/prestashop-1-7.md

## Recurring payments

> The plugin supports [recurring payments](/features/recurring-payments).

Reviewer: ```suggestion
The plugin supports [recurring payments](/docs/recurring-payments).
```

## Refunds

> - [Full and partial refunds](/refunds/) are supported in your MultiSafepay dashboard and backend.  

Reviewer: ```suggestion
- [Full and partial refunds](/docs/refund-payments/) are supported in your MultiSafepay dashboard and backend.  
```

> - [Pay later refunds](/payment-methods/pay-later/) are only supported in your dashboard.

Reviewer: ```suggestion
- [Pay later refunds](/docs/refund-payments#pay-later-refunds) are only supported in your dashboard.
```

## Updates

<details id="how-to-update-in-your-backend">
<summary>How to update in your backend</summary>
<br>

> **Tip!** Make sure you have a backup of your production environment, and that you test the plugin in a staging environment.

1. Download the plugin again above.
2. Follow the [Installation](/docs/prestashop-1-7#how-to-install) instructions.

</details>

## Upgrades

<details id="how-to-upgrade-to-version-5">
<summary>How to upgrade to version 5.x</summary>
<br>

> If you use a verWe recommend upgrade to version 5.x from an older version, follow these steps:

Reviewer: Typo!
